test/unit/compute_resource_host_associator_test.rb

```diff
@@ -9,7 +9,11 @@ class ComputeResourceHostAssociatorTest < ActiveSupport::TestCase
   let(:vm2) { stub('vm2', :identity => Foreman.uuid) }
   let(:vm3) { stub('vm3', :identity => Foreman.uuid) }
   let(:vms) { [vm1, vm2, vm3] }
-  let(:host_with_vm) { FactoryBot.create(:host, :compute_resource => compute_resource) }
+  let(:host_with_vm) do
+    h = FactoryBot.create(:host, :compute_resource => compute_resource)
```

Comment:
```ruby
FactoryBot.create(:host, :compute_resource => compute_resource).tap do |host|
 host.stubs(:vm_exists?).returns(true)
end
```

Reply:
Kinda hate tap, does not bring any improvement in readability, it's line fo line. But let me change that, don't mind.